### DISCUSSION TITLE
What does parserOptions.project do?

### DISCUSSION THREAD

**Discussion by naticaceres** [eslint/eslint#14741] [Q&A] [answered]
I've been browsing the documentation, code, commits and questions here and in stackoverflow looking for an official answer, but found none.
So I ended up here, asking this much needed question.
What is the purpose of the parserOptions.project configuration?
What are the valid values for it?

thanks in advance.

**Comment by mdjermanovic** [comment, marked as answer]
Hi @naticaceres!

ESLint's default parser does not have `parserOptions.project` option.

If you're using `@typescript-eslint/parser`, documentation for the option is here:

https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/parser#parseroptionsproject

**Reply by naticaceres** [reply]
thanks!
sorry lol - I am using typescript-eslint
heading over there now 😅
